scripts/tasks/build-codepen-examples.js

```diff
@@ -1,25 +1,25 @@
+// @ts-check
+
 module.exports = function() {
   const path = require('path');
   const transformer = require('../codepen/codepen-examples-transform');
   const glob = require('glob');
-  const files = glob.sync(path.resolve(__dirname, '../../packages/*/src/components/**/examples/*Example*.tsx'));
-
-  const jscodeshift = require('jscodeshift');
-  const fs = require('fs');
+  const fse = require('fs-extra');
```

Comment:
Sorry if this is obvious, but what is the benefit of `fs-extra` in this change-set? Asking for my own knowledge primarily.

Reply:
Cool, thanks for the explaination @ecraig12345! Noted for my future-self 😄

Reply:
I'm in love with it! I'm switching `just` to this!